src/services/earn/types.ts

```diff
@@ -100,7 +104,7 @@ export type EarnActionSwapConfig = { slippagePercentage?: number; txValidFor?: T
 export type AddFundsEarn = { swapConfig?: EarnActionSwapConfig } & (
   | { permitData: PermitData['permitData']; signature: string }
   | { token: TokenAddress; amount: BigIntish }
-);
+) & { caller: Address };
```

Comment:
Why do we need this here? Don't we have it already as a param for deposit/increase?

Reply:
Yes, i forgot to remove this